instrumentation/lettuce-5.0/src/main/java/io/lettuce/core/AbstractRedisReactiveCommands.java

```diff
@@ -0,0 +1,103 @@
+package io.lettuce.core;
+
+import java.util.function.Consumer;
+import java.util.function.Supplier;
+
+import com.newrelic.api.agent.DatastoreParameters;
+import com.newrelic.api.agent.weaver.Weave;
+import com.newrelic.api.agent.weaver.Weaver;
+import com.nr.fit.lettuce.instrumentation.NRErrorConsumer;
+import com.nr.fit.lettuce.instrumentation.NRHolder;
+import com.nr.fit.lettuce.instrumentation.NRSignalTypeConsumer;
+import com.nr.fit.lettuce.instrumentation.NRSubscribeConsumer;
+
+import io.lettuce.core.api.StatefulConnection;
+import io.lettuce.core.protocol.ProtocolKeyword;
+import io.lettuce.core.protocol.RedisCommand;
+import reactor.core.publisher.Flux;
+import reactor.core.publisher.Mono;
+import reactor.core.publisher.SignalType;
+
+@Weave
+public abstract class AbstractRedisReactiveCommands<K, V>
```

Comment:
Same comment about naming weaved library classed with `_Instrumentation` and specifying the `originalName` in the `Weave` annotation.